sockeye/callback.py

```diff
@@ -243,6 +244,21 @@ def stop_fit_callback(self):
         self._empty_decoder_metric_queue()
         self._write_scores()
 
+    def save_state(self, fname):
+        """
+        Save the state: current metrics and best checkpoint
```

Comment:
add
:param fname: Name of file to save the state to.

Also for the method below